packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -333,7 +337,7 @@ export class Viewer {
    *
    * @private
    */
-  @Event() public sessionidchange!: EventEmitter<string>;
+  @Event() public deviceidchange!: EventEmitter<string>;
```

Comment:
Let's use `deviceIdChange`, we're standardizing on camelcase event names vs the archaic DOM event types that are all lowercased.

Reply:
d33f010